guide/ops/high-availability-supplemental.md

```diff
@@ -0,0 +1,97 @@
+---
+title: High Availability (Supplemental)
+layout: website-normal
+---
+
+This document supplements the High Availability documentation available [here](http://brooklyn.apache.org/v/latest/ops/high-availability.html)
```

Comment:
Would this not be better as a multiple section sub-section similar to locations?